ci/tests/e2e-ansible.sh

```diff
@@ -70,8 +70,11 @@ test_operator() {
         exit 1
     fi
 
+    # todo: remove 7 after the ci prow upgrade
     # verify that the metrics endpoint exists
-    if ! timeout 1m bash -c -- "until kubectl run -i --rm --restart=Never test-metrics --image=registry.access.redhat.com/ubi7/ubi-minimal:latest -- curl -sfo /dev/null http://memcached-operator-metrics:8383/metrics; do sleep 1; done";
+    if [[ ! timeout 1m bash -c -- "until kubectl run -i --rm --restart=Never test-metrics --image=registry.access.redhat.com/ubi7/ubi-minimal:latest -- curl -sfo /dev/null http://memcached-operator-metrics:8383/metrics; do sleep 1; done"
```

Comment:
We don't need to run the metrics tests twice, so I think we can remove the lines that test using ubi7.

All we need is an image that has `curl`, so `ubi8` should work regardless. We typically use the same image here as we do for the operator so that we don't incur another image download, but during the transition it should be okay to just use ubi8 directly.

Ditto for the other tests.

Reply:
Thank for check it Changed as requested 👍